fix(k8s) Improvements to enterprise K8s installation page (#2391)
Kong/docs.konghq.com: app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

    ```
3. Create secret:
    ```
    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
    ```
{% endnavtab %}
{% endnavtabs %}
## Step 7. Prepare Kong's configuration file

> 1. Create a `values.yaml` file for Helm based on the template in the [Kong charts repository](https://github.com/Kong/charts/blob/main/charts/kong/values.yaml). This file contains all the necessary parameters for your Kong deployment. As an example, [this values file](https://github.com/Kong/charts/blob/main/charts/kong/example-values/full-k4k8s-with-kong-enterprise.yaml) enables most Kong Enterprise features.

Reviewer: ```suggestion
1. Create a `values.yaml` file for Helm based on the template in the [Kong charts repository](https://github.com/Kong/charts/blob/main/charts/kong/values.yaml). This file contains all the possible parameters for your Kong deployment. 

    You can also base your configuration on a sample Kong Enterprise `values.yaml` 
    file. For example, [this values file](https://github.com/Kong/charts/blob/main/charts/kong/example-values/full-k4k8s-with-kong-enterprise.yaml) 
    enables most Kong Enterprise features.
```
I'd suggest a better lead-in for the Kong Enterprise sample file, to make it clear why they might want to use it. 

(I hope I got the indents right, it's difficult to judge in these suggestions.)

2. Minimally, for setting up Kong Enterprise on Kubernetes, you will need to set the following parameters:

    |Parameter      | Value                         |
    |---------------|-------------------------------|
    |`enterprise.enabled` | `true` |

>     |`enterprise.portal.enabled` | (Optional) Set to `true` to enable the Kong Developer Portal. Find out more about it [here](https://docs.konghq.com/enterprise/latest/developer-portal/). |

Reviewer: ```suggestion
    |`enterprise.portal.enabled` | (Optional) Set to `true` to enable the [Kong Developer Portal](/enterprise/{{page.kong_version}}/developer-portal/). |
```

    |Parameter      | Value                         |
    |---------------|-------------------------------|
    |`enterprise.enabled` | `true` |

>     |`enterprise.rbac.enabled` | (Optional) Set to `true` to enable RBAC. Requires seeding the super admin password, see above. |

Reviewer: ```suggestion
    |`enterprise.rbac.enabled` | (Optional) Set to `true` to enable RBAC. Requires seeding the super admin password; see [above](#step-5-seed-the-super-admin-password). |
```

    |`env.database` | `"postgres"` or `"cassandra"` |
    |`env.pg_host` | (If using Postgres) Set to host of the Postgres server (only if `postgresql.enabled` is `false`). |
    |`env.pg_port` | (If using Postgres) Set to port of the Postgres server (only if `postgresql.enabled` is `false`). |

>     |`env.pg_user` | (If using Postgres) Set to Postgres user (default `kong`). When `postgresql.enabled` is `true` this has to match `postgresql.postgresqlUsername`. |

Reviewer: ```suggestion
    |`env.pg_user` | (If using Postgres) Set to the Postgres user (default `kong`). When `postgresql.enabled` is `true`, this has to match `postgresql.postgresqlUsername`. |
```

>     |`env.pg_password` | (If using Postgres) Set to Postgres user's password. When `postgresql.enabled` is `true` this has to match `postgresql.postgresqlPassword`. |

Reviewer: ```suggestion
    |`env.pg_password` | (If using Postgres) Set to the Postgres user's password. When `postgresql.enabled` is `true`, this has to match `postgresql.postgresqlPassword`. |
```

>     |`env.pg_database` | (If using Postgres) Set to Postgres database name (default `kong`). When `postgresql.enabled` is `true` this has to match `postgresql.postgresqlDatabase`. |

Reviewer: ```suggestion
    |`env.pg_database` | (If using Postgres) Set to the Postgres database name (default `kong`). When `postgresql.enabled` is `true`, this has to match `postgresql.postgresqlDatabase`. |
```

    |`env.password.valueFrom.secretKeyRef.name` | Name of secret that holds the super admin password. In the example above, this is set to `kong-enterprise-superuser-password`. |
    |`env.password.valueFrom.secretKeyRef.key` | The type of secret key used for authentication. If you followed the default settings in the example above, this is `password`. |
    |`image.repository` | The Docker repository. In this case, `kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition`. |

>     |`image.tag` | The Docker image tag you want to pull down, e.g. `"1.5.0.6-alpine"`. |

Reviewer: ```suggestion
    |`image.tag` | The Docker image tag you want to pull down, e.g. `"{{page.kong_latest.version}}-alpine"`. |
```
Let's make this a variable instead so we don't have to keep updating it.
